runtime/tools/python/ttrt/common/api.py

```diff
@@ -121,6 +121,12 @@ def run(args):
         fbb = ttrt.binary.load_binary_from_path(binary)
         check_version(fbb.version)
         fbb_dict = ttrt.binary.as_dict(fbb)
+        assert (
+            fbb_dict["system_desc"]
+            == system_desc_as_dict(ttrt.runtime.get_current_system_desc()[0])[
```

Comment:
QQ just for me, why do we have multiple system descs in runtime(needing to fetch first for version check)?

Reply:
there's one system desc acquired from the flatbuffer, and the other from the system. Call happens once during runtime for each binary. Is there somewhere else you see it getting called?

Reply:
sync'd offline